src/main/java/com/google/sps/data/PersistentUserRepository.java

```diff
@@ -59,7 +59,6 @@ public User getUser(com.google.appengine.api.users.User googleUser) {
     User newUser = new User(googleUser.getNickname());
     newUser.setEmail(userEmail);
     newUser.setId(googleUser.getUserId());
-    addUser(newUser);
```

Comment:
I removed it bc it kept adding the user when we called this in SessionContext. We don't need to add a user here since it adds it with the profileBuilder.

Reply:
Done